### DISCUSSION TITLE
添加其他输入法

### DISCUSSION THREAD

**Discussion by Mio-Nanami** [fcitx5-android/fcitx5-android#191] [Ideas | 想法]
什么时候加入其他国家的输入法啊，我希望可以直接能够取代Google Gboard,不然还是google输入法好

**Comment by rocka** [comment]
我也希望，但是代码总是要有人来写

**Reply by Pantyhose-X** [reply]
我期待增加键盘语言 [逻辑语](https://mw.lojban.org/index.php?title=Lojban)、[Elefen语](https://elefen.org/)、[Ido语](https://idolinguo.org.uk/whyido.htm)、世界语

**Comment by rocka** [comment]
https://github.com/fcitx5-android/fcitx5-android/releases/tag/0.0.6 发布了，有了日语，其它的语言也可以慢慢加了
